pkg/image/api/v1/types.go

```diff
@@ -308,7 +308,7 @@ type ImageStreamTag struct {
 	Conditions []TagEventCondition `json:"conditions,omitempty" protobuf:"bytes,4,rep,name=conditions"`
 
 	// Image associated with the ImageStream and tag.
-	Image Image `json:"image" protobuf:"bytes,5,opt,name=image"`
+	Image Image `json:"image" patchStrategy:"delete" protobuf:"bytes,5,opt,name=image"`
```

Comment:
I wouldn't expect this to do good things.  The delete directive is for when you building the patch, not for annotating the type, right?  Type annotations are merge or replace as I recall.